tests/test_line.py

```diff
@@ -0,0 +1,121 @@
+import unittest
+from ai_graph_color.line import Line, point_distance
+
+
+class TestLineObject(unittest.TestCase):
+    def test_init_unequal_x(self):
+        """
+        Tests making a line with unequal x values
```

Comment:
Instead of having 
"Tests making a line with unequal x values"
 and 
"Tests making a line with the same x and different y's"

why not have
"Tests making a line with unequal x values"
and
"Tests making a line with unequal y values"

or
"Tests making a line with the same y and different x's"
and
"Tests making a line with the same x and different y's"